src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public static final int RADIUS = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Circle defaultCircle = new Circle(Color.WHITE.name(), RADIUS);
+
+    public Figure getRandomFigure() {
+        int random = new Random().nextInt(5);
```

Comment:
You should not create a new `Random` instance each time you call `getRandomFigure()` method. Use the existing `random` field instead.